lib/ui/text.dart

```diff
@@ -1010,7 +1072,18 @@ class Paragraph extends NativeFieldWrapperClass2 {
   void _layout(double width) native 'Paragraph_layout';
 
   /// Returns a list of text boxes that enclose the given text range.
-  List<TextBox> getBoxesForRange(int start, int end) native 'Paragraph_getRectsForRange';
+  ///
+  /// [boxHeightStyle] and [boxWidthStyle] allow customization of how the boxes
```

Comment:
avoid starting a sentence with a lowercase letter. To avoid this, use something like "The [foo] argument..."